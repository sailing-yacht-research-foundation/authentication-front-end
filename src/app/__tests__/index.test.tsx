import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { App } from '../index';
import MyProvider from 'app/components/Provider';

const renderer = createRenderer();

describe('<App />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(
      <MyProvider>
        <App />
      </MyProvider>);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
