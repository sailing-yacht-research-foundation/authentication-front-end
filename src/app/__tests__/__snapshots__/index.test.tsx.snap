// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render and match the snapshot 1`] = `
<BrowserRouter>
  <Layout
    className="site-layout"
  >
    <Header
      className="site-layout-background"
      style={
        Object {
          "borderBottom": "1px solid #eee",
          "display": "flex",
          "justifyContent": "center",
          "position": "fixed",
          "width": "100%",
          "zIndex": 1,
        }
      }
    >
      <HeaderContent />
    </Header>
    <SideMenu />
    <Content
      className="site-layout-background"
      style={
        Object {
          "margin": "24px 16px",
          "minHeight": 280,
          "padding": 24,
        }
      }
    >
      <Switch>
        <PrivateRoute
          component={[Function]}
          exact={true}
          path="/"
        />
        <Route
          component={[Function]}
          exact={true}
          path="/signin"
        />
        <Route
          component={[Function]}
          exact={true}
          path="/signup"
        />
        <Route
          component={[Function]}
          exact={true}
          path="/verify-account"
        />
        <Route
          component={[Function]}
          exact={true}
          path="/forgot-password"
        />
        <Route
          component={[Function]}
        />
      </Switch>
      <ToastContainer
        autoClose={5000}
        closeButton={[Function]}
        closeOnClick={true}
        draggable={true}
        draggableDirection="x"
        draggablePercent={80}
        hideProgressBar={false}
        newestOnTop={false}
        pauseOnFocusLoss={true}
        pauseOnHover={true}
        position="top-right"
        role="alert"
        rtl={false}
        transition={[Function]}
      />
    </Content>
  </Layout>
  <Memo(GlobalStyleComponent) />
</BrowserRouter>
`;
